tests/TestCase/Network/RequestTest.php

```diff
@@ -169,7 +169,7 @@ public function testQueryStringParsingFromInputUrl()
         $request = new Request(['url' => 'some/path?one=something&two=else']);
         $expected = ['one' => 'something', 'two' => 'else'];
         $this->assertEquals($expected, $request->query);
-        $this->assertEquals('some/path?one=something&two=else', $request->url);
+        $this->assertEquals('some/path', $request->url);
```

Comment:
before this ,`$request->url` returned `some/path` not `some/path?one=something&two=else`
but there is this  test and test tools not able find that?why?